### ISSUE TITLE
Toggle presentation and usage instructions insufficient and inconsistent

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3613]
### Environment
Theme, react storybook and v10

> Operating system
Windows 7 (and all others)

> Browser
n/a

## Detailed description
v10 and Theme shows toggles with and without visible labels.
Storybook shows toggles without visible labels (some even lack the on/off state indicator)

Like any user input, a toggle requires a label. Toggles also benefit signiciantly fromm a text form of the state (on/off), which is present for many but not all of the examples.

The information in the V10 Usage pane states:
> Labels
Use labels with a toggle so the action is clear. Labels should be three words or less and appear on the side of a toggle.

The note on using labels is suitable (altough the location "on the side of a toggle" is inconsistent with virtually every implementation shown as an example). The Usage should also better distinguish between the label and the on/off text which is a textual reinforcement of the visual state indicator.

> What did you expect to happen?
I expect all toggle examples to contain labels, and the documentation to reinforce this requirement and also tackle the on/off text.

> What WCAG 2.1 checkpoint does the issue violate?
[Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions)
[Headings and Labels](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels)

By failing to provice a label, they fail Labels or Instructions. By inferring that the on/off text is a label, the component also essentially fails Headings and Labels

## Additional information

There are many design flaws occurring with Toggles when they are resized, which are exacerbated by this lack of a label and inconsistent use of the on/off text. Almost all these design issues (which will be captured after i complete WCAG failures) are reduced or elminated by the proper use of text.

**Comment by asudoh** [comment]
Hi 👋 if you go to http://react.carbondesignsystem.com/?path=/story/toggle--toggled, click on Knobs tab at the bottom, and enter some text at `labelText` you'll see what you want.
